### ISSUE TITLE
Readme and example config are missing

### ISSUE THREAD

**Issue by FrancescoL96** [krateoplatformops/finops-operator-vm-manager#15]
**Is your feature request related to a problem? Please describe.**
Readme and example config are missing.

**Describe the solution you'd like**
These two files should be added to increase the accessibility of the repository.
